Sources/TensorFlow/Epochs/Collatable.swift

```diff
@@ -35,9 +35,9 @@ extension Collection where Element: Collatable {
   public var collated: Element { .init(collating: self) }
 
   /// Returns the elements of `self`, padded to maximal shape with `padValue`
-  /// and collated.
+  /// and collated. Padding is added at the end unless `atStart` is `true`.
```

Comment:
The summary should be a single sentence fragment, which means you need a blank line between it and any following text.  

The default value for any parameters with defaults needs to be documented.  A `- Parameter atStart:` clause could handle all of this.  If you try really hard, you *might* find a way to fit all the info in the summary, but don't break a sweat doing so!

Reply:
Addressed.